src/channel.js

```diff
@@ -110,6 +117,19 @@ class _Channel {
 
     const geometryArrays = _Channel.deriveImageGeometry(this)
 
+    // Check frame of reference
+    /* To Do: different channels (monochorme images) can have different 
+    FrameOfReferenceUID, what should we do?
+    for the momnet the FrameOfReferenceUID check is disabled
+    -------------------------------
+    if (referenceFrameOfReferenceUID !== this.FrameOfReferenceUID) {
+      throw new Error(
+        'Optical path ' + this.blendingInformation.opticalPathIdentifier +
+        ' image has different FrameOfReferenceUID respect to the reference optical path ' +
+        referenceOpticalPathIdentifier
+      )
+    }*/
+
```

Comment:
yes I agree, now is commented otherwise it will not work with datasets server http://34.68.90.36/ studyInstanceUID = '1.3.6.1.4.1.5962.99.1.2103930081.1286074986.1595536829665.3.0 (and I can't show a demo of the new blending UI in slim). I will not apply/close this comment now, but I will apply it  as soon as we figure out the the grouping issue in https://github.com/MGHComputationalPathology/slim/issues/22.